src/components/ReplyComment/hooks/useReplyComment.js

```diff
@@ -149,7 +151,8 @@ const useReplyComment = ({itemProp, indexFeed, dataFeed, updateParent, page, get
       updateVote: (data, dataVote) => updateVoteParentPostHook(data, dataVote, itemParentProps),
       updateVoteLatestChildren: (data, dataVote) =>
         updateVoteLatestChildrenParentHook(data, dataVote, itemParentProps),
-      getComment: getThisComment
+      getComment: getThisComment,
+      getCommentParent: params.getComment
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new function `getCommentParent` is added to the `itemParentProps` object. Ensure that this function is defined and used correctly.

```diff
+ getCommentParent: params.getComment
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->